pkg/dagger.io/dagger/core/image.cue

```diff
@@ -47,11 +47,24 @@ import (
 	// Target repository address
 	dest: #Ref
 
-	// Filesystem contents to push
-	input: dagger.#FS
-
 	// Container image config
-	config: #ImageConfig
+	config?: #ImageConfig
+
+	// Platform of image
+	platform?: string
+
+	{
+		// Filesystem contents to push
+		input: dagger.#FS
```

Comment:
You could put config in the disjunction too

Reply:
Yes. `platform` could mv to here.

Could we extract to a `core.#Image` ?